@types/swagger-client/index.d.ts

```diff
@@ -37,13 +37,27 @@ declare module 'swagger-client' {
     responses: { [responseStatus: string]: Json };
   }
 
-  export interface Parameter {
+  export type Parameter = ParameterWithSchema | ParameterWithContent;
+
+  interface ParameterBase {
     name: string;
+    in: 'query' | 'header' | 'path' | 'cookie';
```

Comment:
According to the [specification](https://swagger.io/specification/#parameter-object) this field is required so adding it to the interface here.